src/js/tracks/text-track.js

```diff
@@ -41,13 +41,26 @@ const parseCues = function(srcContent, track) {
   };
 
   parser.parse(srcContent);
+
   if (errors.length > 0) {
-    if (console.groupCollapsed) {
-      console.groupCollapsed(`Text Track parsing errors for ${track.src}`);
-    }
-    errors.forEach((error) => log.error(error));
-    if (console.groupEnd) {
-      console.groupEnd();
+    // If WebVTT returns BadSignature (code=0), and no cues were created, check
+    //  if it is an SRT; if so, convert the track from SRT to WebVTT and re-parse
+    if (errors[0].code === 0 && track.cues.length === 0 &&
```

Comment:
For example, here we can (with some work) do something like:
```js
player.trigger('cueparsingerror', {
  callback: function(newContent) {
    parseCues(newContent, track);
  },
  content: srcContent,
  errors: errors
});
```

Then this piece of code will just listen to the `cueparsingerror` on the player, run this code and call the callback. The main potential issue here would be that `player` isn't available and neither is `tech` (yet), so, we'd need to pass in `tech` and then have the player retrigger `cueparsingerror` on itself from the tech.